### PR TITLE
Update installation.md

### PR DESCRIPTION
**Pull request by Heus-Sueh**
adding a flatpak installation directory that was not documented

### CODE REVIEW

**Comment by afonsojramos** [comment]
In which distros is this the case @Heus-Sueh ?

**Comment by Heus-Sueh** [comment]
> In which distros is this the case @Heus-Sueh ?

I believe that in many distros where the user chooses to install Spotify only for their user and not system-wide

per-user flatpak applications are in `$HOME/.local/share/flatpak/app/`

here is the flatpak documentation talking about it: https://github.com/flatpak/flatpak/wiki/Filesystem

File: docs/advanced-usage/installation.md [hunk 1]
```diff
@@ -112,6 +112,10 @@ sudo chmod a+wr /usr/share/spotify/Apps -R
 ```
 /var/lib/flatpak/app/com.spotify.Client/x86_64/stable/active/files/extra/share/spotify/
 ```
+- In some distros it is stored in:
```
Comment (afonsojramos) [inline docs/advanced-usage/installation.md:115]:
```suggestion

- In some distros it is stored in:
```

**Review by afonsojramos — APPROVED** [review]

**Comment by afonsojramos** [comment]
I see, thanks!

**Pull request merged**